string command   # 'RRT' or 'biRRT'
